### ISSUE TITLE
DataTable is reloading after changing dtOptions and change sorting that I set on site.

### ISSUE THREAD

**Issue by algorytmy** [l-lin/angular-datatables#908]
## What versions you are using?

- jquery version: 2.1.4
- datatables version: 1.10.8
- angular-datatables version: 0.5.3

## What's the problem?

When I click on "ID" column title, sorting is changing from asc to desc (asc is default) but when I click on "change options" button, "ID" column is sorted asc (default option). Is possible to don't change sorting after changing options? I tried to read `vm.dtOptions.order` variable and save it as default using `.withOption('order', [columnNumber, 'asc/desc'])` but it doesn't work. Probably clicking on column title don't change `vm.dtOptions.order` variable.

## Can you share your code?

I use code from your site: https://l-lin.github.io/angular-datatables/#/changeOptions

**Comment by l-lin** [comment]
Have you tried using two arrays?

```js
.withOption('order', [[columnNumber, 'asc/desc']])
```

**Comment by algorytmy** [comment]
`.withOption('order', [[columnNumber, 'asc/desc']])` for saving?
It's working with one and two arrays but the question is how to read what column was sorted.

1. First I create datatables with dtOption for example `.withOption('order', [[2, 'asc']])`. 
2. Next I click on third column title and I don't know how to read what column was clicked. 
3. Next I change dtOptions by
`vm.dtOptions.withOption('scrollY', Height + "px");` and here I want to change sorting for the column that I clicked, not default (in this example 3rd not 2nd) so in my opinion i should use `.withOption('order', [[3, 'asc']])` but I don't know how to read this two variables [3, 'asc'].

So the question is how to read what column was sorted(clicked) and what sorting is it (asc or desc)?

(I tried to read `vm.dtOptions.order` before changing options but it read only default sorting.)
